Makefile

```diff
@@ -3,7 +3,11 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+VERSION ?= 3.0.0
+MINOR_VERSION := $(shell v='$(VERSION)'; echo "$${v%.*}")
+
+MAISTRA_REPOSITORY ?= maistra/istio
+MAISTRA_30_COMMIT ?= 16ff353b180a1f461bd380be6ef2d1809b8c1fba
```

Comment:
Why not reference a branch and use the HEAD?

Then you can replace wget&unzip below with a shallow git clone

Reply:
this way we can do both. I would actually like to get away from using the branch name because that created inconsistencies in the past- whenever there was a new commit on maistra/istio, suddenly the istio-operator repo would break until charts were updated. If we're using the commit, that doesn't happen

Reply:
FYI I added the respective prow job, see https://github.com/openshift/release/pull/39722